### ISSUE TITLE
Indicate progress during install

### ISSUE THREAD

**Issue by rgaudin** [kiwix/kiwix-hotspot#140]
In GitLab by @Popolechien on Nov 12, 2017, 07:34

Install is still running but appears stuck at the same step for the past 120+ hours. No timeout indicated, but it would be helpful to have a list counter of sorts indicating bytes transferred (a percentage bar would be less useful and harder to implement I guess).

**Comment by rgaudin** [comment]
In GitLab by @Kelson42 on Nov 12, 2017, 08:27

Can you please publish your last 100 lines of the log?

**Comment by rgaudin** [comment]
In GitLab by @Popolechien on Nov 13, 2017, 11:34

Here we go:
```

TASK [system : Add locales] ****************************************************
changed: [localhost] => (item=fr_FR.UTF-8)
changed: [localhost] => (item=en_US.UTF-8)

TASK [system : Copy a blank rc.local] ******************************************
changed: [localhost]

TASK [system : Copy .bashrc for root] ******************************************
changed: [localhost]

TASK [system : Copy the global vimrc file] *************************************
changed: [localhost]

TASK [system : Copy .bashrc for new username] **********************************
changed: [localhost]

TASK [system : Copy visudo file for ideascube user] ****************************
changed: [localhost]

TASK [system : Create local directory to work from] ****************************
changed: [localhost]

TASK [system : Create ansible hosts directory to work from] ********************
changed: [localhost]

TASK [system : Copy ansible inventory file to client] **************************
ok: [localhost]

TASK [system : Remove the existing localtime file] *****************************
changed: [localhost]

TASK [system : Set a new localtime file corresponding to the timezone] *********
changed: [localhost]

TASK [system : Set timezone variables] *****************************************
changed: [localhost]

TASK [system : Checking if grub conf file is there] ****************************
ok: [localhost]

TASK [system : Shorten GRUB_TIMEOUT] *******************************************
skipping: [localhost]

TASK [nginx : Install nginx web server] ****************************************
changed: [localhost]

TASK [nginx : Remove default nginx vhost config file (if configured).] *********
changed: [localhost]

TASK [nginx : Upload nginx.conf] ***********************************************
changed: [localhost]

TASK [nginx : Copy redirect 301 rules to redirect old kiwix links] *************
changed: [localhost]

TASK [nginx : Enable Virtual host] *********************************************
changed: [localhost]

TASK [uwsgi : Remove uwsgi installed using pip (bug] ***************************
ok: [localhost]

TASK [dnsmasq : include_vars] **************************************************
ok: [localhost]

TASK [dnsmasq : Install dnsmasq] ***********************************************
changed: [localhost]

TASK [dnsmasq : Copy dnsmasq.conf file] ****************************************
changed: [localhost]

TASK [dnsmasq : Copy dnsmasq-spoof.conf file] **********************************
changed: [localhost]

TASK [dnsmasq : Create a new interfaces file] **********************************
changed: [localhost]

TASK [dnsmasq : restart dsnmasq] ***********************************************
changed: [localhost]

TASK [hostapd : Unblock wlan] **************************************************
skipping: [localhost]

TASK [hostapd : Check if hostapd is already installed] *************************
fatal: [localhost]: FAILED! => {"changed": true, "cmd": "dpkg -l | grep hostapd", "delta": "0:00:01.907078", "end": "2017-11-06 23:54:59.737494", "failed": true, "rc": 1, "start": "2017-11-06 23:54:57.830416", "stderr": "", "stdout": "", "stdout_lines": [], "warnings": []}
...ignoring

TASK [hostapd : Install hostapd] ***********************************************
changed: [localhost]

TASK [hostapd : Remove sysVinit startup file] **********************************
changed: [localhost]

TASK [hostapd : include_vars] **************************************************
ok: [localhost]

TASK [hostapd : Copy hostapd.conf file] ****************************************
changed: [localhost]

TASK [hostapd : Copy startup script] *******************************************
changed: [localhost]

TASK [hostapd : Enable service hostapd] ****************************************
changed: [localhost]

TASK [hostapd : Start hostapd] *************************************************
ok: [localhost]

TASK [hostapd : Enable 802.11n] ************************************************
changed: [localhost]

TASK [hostapd : Enable 802.11n] ************************************************
changed: [localhost]

TASK [iptables : Install iptables] *********************************************
ok: [localhost]

TASK [iptables : Authorize routing] ********************************************
changed: [localhost]

TASK [iptables : Copy iptables rules file] *************************************
changed: [localhost]

TASK [iptables : Copy iptables restore file] ***********************************
changed: [localhost]

TASK [iptables : Reload sysctl conf] *******************************************
changed: [localhost]

TASK [network-manager : Install Network-Manager] *******************************
changed: [localhost]

TASK [network-manager : Create entry to ping the KoomBook-doctor] **************
changed: [localhost]

TASK [network-manager : Start KoomBook-doctor script] **************************
changed: [localhost]

TASK [ideascube : include_vars] ************************************************
ok: [localhost]

TASK [ideascube : set_fact] ****************************************************
skipping: [localhost]

TASK [ideascube : debug] *******************************************************
ok: [localhost] => {
    "msg": "Using branding koombook"
}

TASK [ideascube : Ensure the branding directory exists] ************************
changed: [localhost]

TASK [ideascube : Install header-logo branding resources] **********************
changed: [localhost]

TASK [ideascube : Install favicon branding resources] **************************
changed: [localhost]

TASK [ideascube : include_vars] ************************************************
ok: [localhost]

TASK [ideascube : Configure the ideascube package repository] ******************
changed: [localhost]

TASK [ideascube : Install ideascube] *******************************************
changed: [localhost]

TASK [ideascube : Verify if ideascube folder does exist on system file] ********
skipping: [localhost]

TASK [ideascube : Remove existing folder on external hard drive] ***************
skipping: [localhost]

TASK [ideascube : Move ideascube folder to the hard drive] *********************
skipping: [localhost]

TASK [ideascube : Symlink the ideascube folder from hard drive to sd card] *****
skipping: [localhost]

TASK [ideascube : Ensure the folder is removed] ********************************
skipping: [localhost]

TASK [ideascube : Create ideascube cache folder on hard drive] *****************
skipping: [localhost]

TASK [ideascube : Create a simlink for ideascube catalog] **********************
skipping: [localhost]

TASK [ideascube : Run Ideascube migrations after the first installation] *******
changed: [localhost]

TASK [ideascube : Set IDEASCUBE_ID variable for koombook] **********************
ok: [localhost]

TASK [ideascube : Change IDEASCUBE_ID] *****************************************
changed: [localhost]

TASK [ideascube : Set Ideascube name] ******************************************
changed: [localhost]

TASK [ideascube : Get ideascube version] ***************************************
changed: [localhost]

TASK [ideascube : debug] *******************************************************
ok: [localhost] => {
    "msg": "ideascube 0.29.2 is now the current version."
}

TASK [ideascube : Enable the Ideascube Nginx site] *****************************
changed: [localhost]

TASK [ideascube : Enable the Ideascube uWSGI app] ******************************
changed: [localhost]

TASK [ideascube : Copy ideascube_leftover_files.service systemd unit] **********
changed: [localhost]

TASK [ideascube : Copy ideascube_leftover_files.timer systemd unit] ************
changed: [localhost]

TASK [ideascube : Enable unit ideascube_leftover_files.timer] ******************
changed: [localhost]

TASK [ideascube : Copy remove_staticfiles_json.sh] *****************************
changed: [localhost]

TASK [ideascube : Copy remove_staticfiles_json.service] ************************
changed: [localhost]

TASK [ideascube : Enable unit remove_staticfiles_json.service] *****************
ok: [localhost]

TASK [ideascube : ini_file] ****************************************************
changed: [localhost]

TASK [ideascube : reload ansible_local] ****************************************
ok: [localhost]

TASK [ideascube : Push file to central_server] *********************************
skipping: [localhost]

TASK [kiwix : include_vars] ****************************************************
ok: [localhost]

TASK [kiwix : Get kiwix_version from device_list if exist] *********************
skipping: [localhost]

TASK [kiwix : debug] ***********************************************************
ok: [localhost] => {
    "msg": "kiwix_version 1.0"
}

TASK [kiwix : Copying kiwix binaries x86_64 architecture] **********************
skipping: [localhost]

TASK [kiwix : Copying kiwix binaries v1.0 for armhf architecture] **************
changed: [localhost]

TASK [kiwix : Copying kiwix binaries v0.9 for armhf architecture] **************
skipping: [localhost]

TASK [kiwix : Create a kiwix directory in ideascube folder] ********************
changed: [localhost]

TASK [kiwix : Setup startup file] **********************************************
changed: [localhost]

TASK [kiwix : Remove nginx configuration otherwise, if hostname var change it won't be overrided] ***
ok: [localhost]

TASK [kiwix : Copy nginx vhost] ************************************************
changed: [localhost]

TASK [kiwix : Remove old OSMybon quick fix] ************************************
fatal: [localhost]: FAILED! => {"changed": true, "cmd": ["ideascube", "catalog", "remotes", "remove", "OSMybon"], "delta": "0:00:51.459984", "end": "2017-11-07 00:25:32.830843", "failed": true, "rc": 1, "start": "2017-11-07 00:24:41.370859", "stderr": "CommandError: There is no \"OSMybon\" remote", "stdout": "\u001b[36mIDEASCUBE_ID=kb\u001b[0m\n\u001b[36mImporting settings from ideascube.conf.kb\n\u001b[0m", "stdout_lines": ["\u001b[36mIDEASCUBE_ID=kb\u001b[0m", "\u001b[36mImporting settings from ideascube.conf.kb", "\u001b[0m"], "warnings": []}
...ignoring

TASK [kiwix : Install catalogs] ************************************************
changed: [localhost] => (item={u'url': u'http://catalog.ideascube.org/kiwix.yml', u'name': u'Kiwix', u'description': u'Kiwix ZIM content'})
changed: [localhost] => (item={u'url': u'http://catalog.ideascube.org/static-sites.yml', u'name': u'StaticSites', u'description': u'Static sites'})
changed: [localhost] => (item={u'url': u'http://catalog.ideascube.org/osmaps.yml', u'name': u'OSMybon', u'description': u'Experimental OSM catalog'})
changed: [localhost] => (item={u'url': u'http://catalog.ideascube.org/omeka.yml', u'name': u'Omeka', u'description': u'Omeka packages'})
changed: [localhost] => (item={u'url': u'http://catalog.ideascube.org/bibliotecamovil.yml', u'name': u'bibliotecamovil', u'description': u'Spanish packages'})

TASK [kiwix : Nginx enable Virtual host] ***************************************
changed: [localhost]

TASK [kiwix : Enable service kiwix] ********************************************
changed: [localhost]

TASK [kiwix : ini_file] ********************************************************
changed: [localhost]

TASK [kiwix : reload ansible_local] ********************************************
ok: [localhost]

TASK [kiwix : Push file to central_server] *************************************
skipping: [localhost]

TASK [package_management : include_vars] ***************************************
ok: [localhost]

TASK [package_management : Update ideascube catalog cache before downloading] **
changed: [localhost]

TASK [package_management : Install, upgrade or remove a package present in device_list.fact] ***
changed: [localhost] => (item={u'status': u'present', u'name': u'gutenberg.fr'})
```

**Comment by rgaudin** [comment]
In GitLab by @Kelson42 on Nov 14, 2017, 06:43

@mgautierfr @thiolliere Any idea what goes wrong here?

**Comment by rgaudin** [comment]
In GitLab by @Kelson42 on Nov 14, 2017, 06:45

@Popolechien To me it sounds you should have wait.... Can you please rename the ticket, it sounds otherwise pretty much like a duplicate of #79

**Comment by rgaudin** [comment]
In GitLab by @Popolechien on Nov 14, 2017, 06:46

@Kelson42 Well it's been like that (this exact stage) for 150 hours... How much longer should I have waited?

**Comment by rgaudin** [comment]
In GitLab by @Kelson42 on Nov 14, 2017, 06:58

@Popolechien ah ok... a bit different indeed...

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on Jul 24, 2018, 15:14

mentioned in merge request !49

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on Jul 24, 2018, 21:54

mentioned in commit adc02179ab5e64879134561f463138a9d2606f69

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on Jul 24, 2018, 21:54

closed via merge request !49
